run_2000.py

```diff
@@ -0,0 +1,14 @@
+import time
+
+
+def timer(f, *args, **kwargs):
+    start_time = time.time()
```

Comment:
time() is not an accurate enough method in order to measure the timings of simple python functions. search the time module documentation for a more accurate function.